Mathlib/Combinatorics/SimpleGraph/Trails.lean

```diff
@@ -164,9 +164,8 @@ theorem IsEulerian.card_filter_odd_degree [Fintype V] [DecidableRel G.Adj] {u v
 #align simple_graph.walk.is_eulerian.card_filter_odd_degree SimpleGraph.Walk.IsEulerian.card_filter_odd_degree
 
 theorem IsEulerian.card_odd_degree [Fintype V] [DecidableRel G.Adj] {u v : V} {p : G.Walk u v}
-    (ht : p.IsEulerian) :
-    Fintype.card { v : V | Odd (G.degree v) } = 0 ∨ Fintype.card { v : V | Odd (G.degree v) } = 2 :=
-  by
+    (ht : p.IsEulerian) : Fintype.card { v : V | Odd (G.degree v) } = 0 ∨
+    Fintype.card { v : V | Odd (G.degree v) } = 2 := by
```

Comment:
```suggestion
    (ht : p.IsEulerian) : Fintype.card { v : V | Odd (G.degree v) } = 0 ∨
      Fintype.card { v : V | Odd (G.degree v) } = 2 := by
```
the general rule is, whenever a group (the return type in this case) is split on multiple lines and there are sibling groups on one of those lines (the last parameter), you should either have an additional line just for the group, or you should use a hanging indentation for subsequent lines. The purpose is to avoid the return type from being misread as just `Fintype.card { v : V | Odd (G.degree v) } = 2` in this example.